homu/server.py

```diff
@@ -242,7 +255,7 @@ def github():
     payload = request.body.read()
     info = request.json
 
-    lazy_debug(logger, lambda: 'info: {}'.format(utils.remove_url_keys_from_json(info)))
+    lazy_debug(logger, lambda: 'info: {}'.format(utils.remove_url_keys_from_json(info)))  # NOQA
```

Comment:
Since we're adding the #noqa comments, can you update the Travis file to prevent regressions by adding `flake8 homu/server.py`?